woocommerce-gateway-ebanx/includes/class-wc-ebanx-oxxo-gateway.php

```diff
@@ -73,6 +73,7 @@ protected function save_order_meta_fields($order, $request)
 		parent::save_order_meta_fields($order, $request);
 
 		update_post_meta($order->id, '_oxxo_url', $request->payment->oxxo_url);
```

Comment:
Every time we use `update_post_meta`, the second parameter always need to start with `_ebanx` prefix. In this case, the right way is: `update_post_meta($order->id, '_ebanx_oxxo_url', $request->payment->oxxo_url);`.

Can you update the two updates (`_payment_due_date` to `_ebanx_payment_due_date`) and update the another places that use this meta? 

Thank you!